paimon-web-server/src/main/resources/i18n/messages.properties

```
@@ -35,7 +35,16 @@ menu.name.exist=This menu name is exist:{0}
 menu.path.invalid=This menu path is invalid:{0}
 catalog.name.exist=This catalog name {0} is exist
 catalog.create.error=An exception is returned when calling the Paimon API to create a Catalog.
+catalog.remove.error=An exception is returned when calling the Paimon API to remove a Catalog.
```

```suggestion
catalog.remove.error=Exception calling Paimon Catalog API to remove a Catalog.
```
The below could update with this suggestion.